### ISSUE TITLE
KDE plasma is a bit unstable

### ISSUE THREAD

**Issue by martinbone** [armsurvivors/armbian-release#1]
Yesterday I tried this image for the OP5:

[Armbian-unofficial_24.03.10-armsurvivors-15_Orangepi5_jammy_vendor_6.1.43_kde-neon-amazingfated-edk2_desktop.img.xz](https://github.com/armsurvivors/armbian-release/releases/download/24.03.10-armsurvivors-15/Armbian-unofficial_24.03.10-armsurvivors-15_Orangepi5_jammy_vendor_6.1.43_kde-neon-amazingfated-edk2_desktop.img.xz)

It seems really snappy and promising.
You're probably aware of this already, but I found KDE plasma (or part of it) unstable, and the task bar / menu stops working quite often, and I have to reboot.

**Comment by rpardini** [comment]
Yes. That's a testing/development build of every single component included, KDE included -- I'd be surprised if it was stable. Also wondering how you found those images...

**Comment by martinbone** [comment]
> Yes. That's a testing/development build of every single component included, KDE included -- I'd be surprised if it was stable. Also wondering how you found those images...

Ok. 👍. I noticed you mentioning KDE builds on Discord, so did a Google / GitHub search ...

**Comment by rpardini** [comment]
Nice. What about edk2?
